### PR TITLE
[B2BP-639] - Fix style Header

### PR DESCRIPTION
**Pull request by Meolocious**
#### List of Changes
- Fixed header Logo height
- Updated Storybook from 8.1.0 to 8.1.4

#### Motivation and Context
This PR is made to align components to the legacy version of SEND

#### How Has This Been Tested?
Tested locally

#### Types of changes
- [ ] Chore (nothing changes by a user perspective)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] Doesn't need any documentation update.

### CODE REVIEW

File: apps/nextjs-website/react-components/components/Header/Header.helpers.tsx [hunk 1]
```diff
@@ -20,9 +22,14 @@ export const DialogBubble = ({
 }: DialogBubbleProps) => {
   const muiTheme = useTheme();
   const styles = useStyles(muiTheme);
+  
   return (
     <Stack
-      sx={{ ...styles.bubbleContainer, bgcolor: muiTheme.palette.common.white }}
+      sx={{ 
+        ...styles.bubbleContainer, 
+        bgcolor: muiTheme.palette.common.white,
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Header/Header.helpers.tsx:30]:
```suggestion
        bgcolor: 'common.white', 
```
We can probably use theme-aware properties

File: apps/nextjs-website/react-components/components/Header/Header.tsx [hunk 2]
```diff
@@ -54,15 +54,15 @@ const Header = ({
...
       component='header'
       role='banner'
+      sx={{ height: { xs: 'auto', md: 130 } }}
     >
       <Stack
         direction={{ md: 'row' }}
-        paddingY={{ xs: 2, sm: 3, md: 0 }}
-        gap={4}
+        paddingY={{ xs: 1, sm: 3, md: 0 }}
+        gap={{ xs: 2, md: 2 }}
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Header/Header.tsx:64]:
```suggestion
        gap={2}
```
Is this not the same?

File: apps/nextjs-website/react-components/components/Header/Header.tsx [hunk 3]
```diff
@@ -54,15 +54,15 @@ const Header = ({
...
+        style={{ height:'100%' }}
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Header/Header.tsx:65]:
```suggestion
        sx={{ height:'100%' }}
```
I think it's better to use 'sx' when working with MUI and for consistency

File: apps/nextjs-website/react-components/components/Header/Header.tsx [hunk 4]
```diff
@@ -80,12 +80,15 @@ const Header = ({
           />
 
           <Stack
-            sx={{ display: { md: 'none' } }}
+            sx={{ 
+              display: { md: 'none' },
+              marginRight: { xs: 4, md: 0 }, // Add this line
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Header/Header.tsx:85]:
```suggestion
              marginRight: { xs: 4, md: 0 },
```
Useless comment

File: apps/nextjs-website/react-components/components/Header/components/MenuDropdown.tsx [hunk 5]
```diff
@@ -88,66 +79,92 @@ export const MenuDropdown = (props: MenuDropdownProp) => {
...
+        <DropdownParent>
+          {hasLinks && dropdownVisible && (
+            <Dropdown gap={1}>
+              {items?.map((item: DropdownItem, index) => (
+                <Link
+                  variant="body1"
+                  underline="none"
+                  key={item.key ?? index}
+                  sx={styles.link}
+                  style={{ color: active ? muiTheme.palette.primary.dark : muiTheme.palette.text.secondary, textDecoration: 'none', fontSize: '1em', fontWeight: 600 }}
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Header/components/MenuDropdown.tsx:159]: Why separate sx and style?

File: apps/nextjs-website/react-components/components/Header/components/MenuDropdown.tsx [hunk 6]
```diff
@@ -88,66 +79,92 @@ export const MenuDropdown = (props: MenuDropdownProp) => {
...
+        minWidth:'max-content', 
+        height:'100%', 
+        gap: { xs: 1, md: 1.5 }, 
+        display:'flex', 
+        flexDirection:'row', 
+        justifyContent: { xs: 'left', md: 'center' },
+        alignItems: { xs: 'center', md: 'center' },
+        paddingLeft: { xs: 2, md: 0 },
+      }}>
+        <Link sx={{ justifyContent: { xs: 'left', md: 'center' }, alignContent: { xs: 'left', md: 'center' }}} style={{ color: active ? muiTheme.palette.primary.dark : muiTheme.palette.text.secondary, textDecoration: 'none' }} {...button}>
```
Comment (simosalsi) [inline apps/nextjs-website/react-components/components/Header/components/MenuDropdown.tsx:131]: Why separate sx and style?

File: apps/nextjs-website/src/app/theme.ts [hunk 7]
```diff
@@ -21,6 +21,18 @@ declare module '@mui/material/styles' {
...
+      readonly boxShadow?: string;
+      readonly otMenuMobile?: string;
+    };
+  }
+  interface ShadowsOptions {
+    readonly custom?: {
+      readonly boxShadow?: string;
+      readonly otMenuMobile?: string;
+    };
+  }
```
Comment (simosalsi) [inline apps/nextjs-website/src/app/theme.ts:35]:
```suggestion
```
I believe this doesn't work for Shadows and is hence useless

**Review by simosalsi — CHANGES_REQUESTED** [review]

**Review by simosalsi — APPROVED** [review]

**Review by maurodandrea — APPROVED** [review]

**Pull request merged**